internal/cmdinit/cmdinit.go

```diff
@@ -104,6 +105,10 @@ func (r *Runner) runE(c *cobra.Command, args []string) error {
 					NameMeta: yaml.NameMeta{
 						Name: r.Name,
 					},
+					// mark Kptfile as local-config
+					Annotations: map[string]string{
+						filters.LocalConfigAnnotation: "true",
```

Comment:
we also have a local-config const var in our new sdk
`fn.KptLocalConfig`

Reply:
There is other code in kpt (for ex. `kpt live`) that also uses this annotation. We can switch to using new const in all places as separate effort.